docs/api/web-contents.md

```diff
@@ -1081,6 +1114,17 @@ Works like `executeJavaScript` but evaluates `scripts` in an isolated context.
 
 Ignore application menu shortcuts while this web contents is focused.
 
+#### `contents.setWindowOpenOverride(handler)`
```

Comment:
`setWindowOpenHandler`? You've even called the parameter `handler` so I think it fits a bit better

Reply:
I think I agree with you, but will let the API WG weigh in before making the change.